hydrogym/firedrake/envs/pinball/flow.py

```diff
@@ -13,6 +13,36 @@
 
 
 class Pinball(FlowConfig):
+  rad = 0.5
```

Comment:
`rad` is already defined below, it could be good to pass an unused variables check (shipped with standard static checkers)